source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -266,7 +266,8 @@ photos of the same product from different angles or with a different
 presentation or combinations of the above add value to the presentation
 of the product in <%= settings.site_name.capitalize %>.
 
-This field is optional but recommended. It can occur multiple times
+This field is optional but recommended, except of fashion categories
```

Comment:
`except for`

https://www.usingenglish.com/forum/threads/198474-except-of

Reply:
amr @Zorbash 👑

Reply:
fixed